scipy/differentiate/tests/test_differentiate.py

```diff
@@ -473,15 +472,21 @@ def test_iv(self):
             jh_func(func, x, maxiter=-1)
 
 
+@pytest.mark.skip_xp_backends('array_api_strict',
+                              reason='Currently uses fancy indexing assignment.')
+@pytest.mark.skip_xp_backends('jax.numpy',
+                              reason='JAX arrays do not support item assignment.')
+@pytest.mark.skip_xp_backends('cupy',
+                              reason='cupy/cupy#8391')
```

Comment:
It looks like cupy/cupy#8391 was closed before the release of  [CuPy13.3.0](https://github.com/cupy/cupy/releases/tag/v13.3.0). Are we testing with the latest version?